plugins/modules/ssh_config.py

```diff
@@ -108,6 +108,19 @@
       - Sets the C(HostKeyAlgorithms) option.
     type: str
     version_added: 6.1.0
+  controlmaster:
+    description:
+      - Sets the C(ControlMaster) option.
+    choices: [ 'yes', 'no', 'ask', 'auto', 'autoask' ]
+    type: str
```

Comment:
Same for the other new options.
```suggestion
    type: str
    version_added: 8.1.0
```

Reply:
Done.